src/releasers/ocaml.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ReleasePR, ReleaseCandidate} from '../release-pr';
+
+import {ConventionalCommits} from '../conventional-commits';
+import {GitHubTag} from '../github';
+import {checkpoint, CheckpointType} from '../util/checkpoint';
+import {Update} from '../updaters/update';
+import {Commit} from '../graphql-to-commits';
+
+// Generic
+import {Changelog} from '../updaters/changelog';
+// OCaml
+import {Opam} from '../updaters/ocaml/opam';
+import {EsyJson} from '../updaters/ocaml/esy-json';
+
+const notEsyLock = (path: string) => !path.startsWith('esy.lock');
+
+const CHANGELOG_SECTIONS = [
+  {type: 'feat', section: 'Features'},
+  {type: 'fix', section: 'Bug Fixes'},
+  {type: 'perf', section: 'Performance Improvements'},
+  {type: 'revert', section: 'Reverts'},
+  {type: 'docs', section: 'Documentation'},
+  {type: 'chore', section: 'Miscellaneous Chores'},
+  {type: 'refactor', section: 'Code Refactoring'},
+  {type: 'test', section: 'Tests'},
```

Comment:
Is the goal here to show additional categories of PRs in CHANGELOG, do you think wanting to show these additional sections will be common in the OCaml community?

My concern is that showing every docs change, chore, etc., can get a little noisy.

Reply:
The common convention in the ocaml community is to show all of this. Ideally, these would be configurable by consumers and just provide a sensible default.